test/unittests/test_intx.cpp

```diff
@@ -490,4 +490,18 @@ TYPED_TEST(uint_test, to_string_base)
     EXPECT_EQ(to_string(x, 36), "sg");
     EXPECT_EQ(to_string(x, 2), "10000000000");
     EXPECT_EQ(to_string(x, 8), "2000");
-}
\ No newline at end of file
+}
+
+TYPED_TEST(uint_test, as_bytes)
+{
+    constexpr auto x = TypeParam{0xa05};
+    const auto b = as_bytes(x);
+    EXPECT_EQ(b[0], 5);
+    EXPECT_EQ(b[1], 0xa);
```

Comment:
This codifies it as little endian. Should this be implemented for both be/le ?

Reply:
I have to add big endian support in general. But not a priority in the moment. I only have time to add features needed in other EVM projects.

Reply:
Well EVM is big endian, isn't it?

Reply:
Correct, but this API is needed for `intx::be` API and in one place in evmone. It exposes internal representation as bytes.

Reply:
So perhaps can you move `as_bytes` into `intx::be` ?